### ISSUE TITLE
Allow commands to be completly disabled, so that the don't consume any resources when disabled.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#287]
```
Hey there,
Love the app and how advanced it is but memory usage is about 5 times higher 
than my other background apps and it's affecting battery life.

[1] The TTS process is running when I have no tts options checked and have 
never used those fcns.
[2] Could part of the process be cached instead of active?
[3] Being such a powerful app you should look into the potential of using 
gtalksms with Tasker/Locale or IFTTT.

Thanks in advance,
Mike
Nexus S on Jellybean
```

Original issue reported on code.google.com by `mike.ra...@gmail.com` on 10 Aug 2012 at 6:38

**Comment by GoogleCodeExporter** [comment]
```
I totally agree. The big memory footprint was one thing I discovered since the 
4.0 release. The TTS engine beeing one of the greatest consumers.

With the new command manager in place I think it should be possible to disable 
commands entirely, so that they don't acquire any resource when disabled. This 
would need some code changes, but I think that's the right way to go and 
manageable. On the long-run we could even think about modularizing GTalkSMS in 
many .apks.
```

Original comment by `fschm...@gmail.com` on 11 Aug 2012 at 9:14
- Changed title: **Allow commands to be completly disabled, so that the don't consume any resources when disabled.**
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 20 Aug 2012 at 9:40
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Code is now commited, we should test all commands prior a new play store 
release.
```

Original comment by `fschm...@gmail.com` on 28 Aug 2012 at 12:40
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Awesome, thanks!
```

Original comment by `mike.ra...@gmail.com` on 30 Aug 2012 at 5:12

**Comment by GoogleCodeExporter** [comment]
```
We just released a new beta which contains the fix for this issue. If you are 
advantageous, grab the apk from http://code.google.com/p/gtalksms/downloads/list
```

Original comment by `fschm...@gmail.com` on 10 Sep 2012 at 9:56
- Added labels: **BetaWithFixAvailable**

**Comment by GoogleCodeExporter** [comment]
```
Will be release with GTalkSMS 4.3
```

Original comment by `fschm...@gmail.com` on 11 Nov 2012 at 11:21
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 4.3 release
```

Original comment by `fschm...@gmail.com` on 26 Jan 2013 at 5:45
- Changed state: **Deployed**
